EventsExpress/ClientApp/src/actions/alert-action.js

```diff
@@ -27,6 +27,13 @@ export function setSuccessAllert(msg) {
     }
 }
 
+export function setErrorAllert(msg) {
```

Comment:
Misspelled allert = should be `alert`.

Reply:
It is codestyle.
![image](https://user-images.githubusercontent.com/59233444/114538371-fd444980-9c5b-11eb-942e-a367d659d8c9.png)